Makefile

```diff
@@ -11,6 +11,6 @@ lint:
 	pipenv run pylint --output-format=colorized -j 0 --reports=n ./secure_message
 	pipenv check ./secure_message ./tests
 
-test: lint
```

Comment:
why are we not running the lint? on make test

Reply:
Accidental push